functions/load_metric.py

```diff
@@ -0,0 +1,129 @@
+import boto3
+import datetime
+import logging
+import os
+
+cloudwatch = boto3.client("cloudwatch")
+logger = logging.getLogger()
+logger.setLevel(os.environ.get("LOGLEVEL", logging.INFO))
+
+
+def handler(event, context):
+    logger.debug(f"environment variables:\n{os.environ}")
+    before, latest = get_period_timestamps()
+    metrics = get_metrics(before, latest)
+    logger.debug(f"available metrics for {before}~{latest}:\n{metrics}")
+
+    ANOMV = metrics["maxANOMV"][latest]
+    NOER = metrics["sumNOER"][latest]
+    GISI = metrics["avgGISI"][latest]
```

Comment:
Still missing the boot discount

Reply:
I think we should try to make the formula more accurate through more accurate metrics (#143) instead of making approximations.